libraries/botbuilder-dialogs-declarative/tests/resources/TestFolder/test.schema

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
This file was duplicated elsewhere and not used anywhere. The duplication caused an issue when running `bf merge`